### PR TITLE
[Behat] Move extensions and services

### PR DESCRIPTION
**Pull request by tuka217**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT
| Doc PR          | Sylius/Sylius-Docs#X

### CODE REVIEW

File: src/Sylius/Behat/Page/Admin/Crud/IndexPage.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 use Behat\Mink\Session;
 use Sylius\Behat\Page\ElementNotFoundException;
 use Sylius\Behat\Page\SymfonyPage;
-use Sylius\Behat\TableManipulatorInterface;
+use Sylius\Behat\Service\Accessor\TableManipulatorInterface;
```
Comment (lchrusciel) [inline src/Sylius/Behat/Page/Admin/Crud/IndexPage.php]: If we decide to move `TableManipulatorInterface` to `Accessor` namespace can we also rename it to `TableAccessorInterface` whit rest of related classes and calls according to #4511?

Comment (tuka217) [inline src/Sylius/Behat/Page/Admin/Crud/IndexPage.php]: Yes, no problem.

**Comment by lchrusciel** [comment]
After green build it is probably good to go :+1: 
/cc @pjedrzejewski

**Comment by pjedrzejewski** [comment]
Thank you Ania!

**Pull request merged**